cypress/e2e/CMS/$Article/index.js

```diff
@@ -0,0 +1,152 @@
+    cy.getNthIframe("iframe[title='Rich Text Editor, Heading field']", position -1).find('p').type(value)
+})
+
+And('user fills out {int} {string} text area with {string}', (position, sectionName, value) => {
+    cy.getNthIframe("iframe[title='Rich Text Editor, Content field']",position - 1).find('p').type(value)  
+})
+
+And('intro text reads {string}', (titlText) => {
+    cy.get(`div[class='blog-intro-text'] p:contains("${titlText}")`).should('be.visible')
+})
+
+And('{int} section heading reads {string}', (num, title) => {
+    cy.get(`h2[id='article-heading1'] p:contains('${title}')`).should('be.visible')
+})
+
+And('{int} description reads {string}', (contentText) => {
+    cy.get(`div[class='medium-9 columns contentzone has-section-nav'] p:contains('${contentText}')`).should('be.visible')
+})
+
+And('public use text has a link {string} with href {string}', (linkText, linkUrl) => {
+    cy.get(`div[class='public-use'] a:contains('${linkText}')`).should('be.visible').and('have.attr', 'href', linkUrl)
+})
+
+And('{string} section is not displayed', (section) => {
+    cy.get(`a:contains("${section}")`).should('not.exist')
+})
+
+let imageSrc;
+And('user selects {int} Lead Image from the list of images', (num) => {
+    cy.get('span:contains("Lead Image")').parent().as('imageUpload').click()
+    cy.get('input[name="field_image_article_entity_browser_entity_browser"]').click({ force: true })
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id^='edit-entity-browser-select-media']").eq(num - 1).check()
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id='edit-submit'][value='Select image']").click({ force: true })
+}); 
+And('user remembers the source of selected lead image for further verification', () => {
+    cy.get('details img').then($el => {
+        imageSrc = $el[0].getAttribute('src')
+    })
+})
+
+let imageSrc1;
+And('user selects {int} Promotional Image from the list of images', (num) => {
+    cy.get('span:contains("Promotional Image")').parent().as('imageUpload').click()
+    cy.get('input[name="field_image_promotional_entity_browser_entity_browser"]').click({ force: true })
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id^='edit-entity-browser-select-media']").eq(num - 1).check()
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id='edit-submit'][value='Select image']").click({ force: true })
+})
+
+And('user remembers the source of selected promo image for further verification', () => {
+    cy.get('details img').then($el => {
+        imageSrc1 = $el[0].getAttribute('src')
+    })
+})
+
+And('user selects {string} checkbox', (dateDisplay) => {
+    cy.get(`[class='fieldset-wrapper'] label:contains("${dateDisplay}")`).parent().find('input.form-checkbox').check({ force: true })
+})
+
+And("{string} date is displaying today's date", (date) => {
+    cy.get('ul.clearfix li strong').should('be.visible').and('include.text', date)
```

Comment:
this step is not verifying the actual date - you need to extract the text from html date tag and compare it with today's date ( which you get from a Date object, similar logic to https://github.com/NCIOCPL/cgov-digital-platform-acceptance-tests/blob/67b045aae74524cd9a93adbe60b4d5f3496a852f/cypress/e2e/common/cmsSteps.js#L159 )